resources/qml/PrintSetupTooltip.qml

```diff
@@ -17,6 +17,8 @@ UM.PointingRectangle
     arrowSize: UM.Theme.getSize("default_arrow").width
 
     opacity: 0
+    // This should be disabled when invisible, otherwise it will catch mouse events.
+    enabled: false
```

Comment:
Alternatively, you could simply bind enabled and opacity together, like so:

`enabled: opacity != 0`

This makes the other two lines superfluous. I like one-line-fixes ;-)